controls/sql.sp

```diff
@@ -0,0 +1,44 @@
+locals {
+  sql_common_tags = merge(local.thrifty_common_tags, {
+    service = "sql"
+  })
+}
+
+benchmark "sql" {
+  title         = "SQL Checks"
+  description   = "Thrifty developers checks SQL old databases."
+  documentation = file("./controls/docs/sql.md")
+  tags          = local.sql_common_tags
+  children = [
+    control.sql_database_age_90,
+  ]
+}
+
+control "sql_database_age_90" {
+  title       = "Old SQL database should be reviewed"
```

Comment:
@khushboo9024 Can you please update the title/description to match the one for snapshots in Compute?